db/schema.rb

```diff
@@ -1,4 +1,3 @@
-# encoding: UTF-8
```

Comment:
How did this get removed? On my machine, doing a `rake db:migrate` adds it back.